library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -87,7 +98,8 @@ public void initialize(FragmentManager fragmentManager) {
     if (mapInterface != null) {
       initialize(fragmentManager, mapInterface);
     } else {
-      initialize(fragmentManager, new DefaultAirMapViewBuilder(getContext()).builder().build());
+      AirMapViewBuilder builder = new DefaultAirMapViewBuilder(getContext()).builder();
+      initialize(fragmentManager, builder.build());
```

Comment:
we can revert these 2 lines, since we no longer need a reference to the `builder`